magenta/models/nsynth/wavenet/fastgen.py

```diff
@@ -83,7 +86,7 @@ def load_fastgen_nsynth(batch_size=1):
   return graph
 
 
-def encode(wav_data, checkpoint_path, sample_length=64000):
+def encode(wav_data, checkpoint_path, FLAGS=None, sample_length=64000):
```

Comment:
For consistency we reserve the all caps FLAGS for global variables, so maybe change it to `flags`, or something more descriptive like `cpu_optimizations`

Reply:
did it

Reply:
I changed to flags.